src/main/scala/strawman/collection/Factories.scala

```diff
@@ -73,6 +43,7 @@ trait IterableFactory[+CC[_]] {
   def apply[A](xs: A*): CC[A] = fromIterable(View.Elems(xs: _*))
   def fill[A](n: Int)(elem: => A): CC[A] = fromIterable(View.Fill(n)(elem))
   def newBuilder[A](): Builder[A, CC[A]]
+  implicit def canBuildIterable[A]: FromSpecificIterable[A, CC[A]] = IterableFactory.toSpecific(this)
```

Comment:
BTW, I’d like to rename `FromSpecificIterable[A, C]` to `CanBuild[A, C]`. If users write code that takes parameters of that type, I think `CanBuild` conveys a clearer meaning (“ability to build a collection `C` from elements of type `A`”) than `FromSpecificIterable`. WDYT?